docs/int/openapi.yaml

```diff
@@ -1106,7 +1296,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/JobCode'
-
+    MetaDataPatch:
```

Comment:
Should be 'MetadataListUpdate' to match other component schemas and place this schema alphabetically in the components section, per current formatting.

- [ ] rename schema